sdk/src/main/java/io/opentdf/platform/sdk/SDKBuilder.java

```diff
@@ -135,12 +190,22 @@ public SDK build() {
     }
 
     private ManagedChannelBuilder<?> getManagedChannelBuilder() {
-        ManagedChannelBuilder<?> channelBuilder = ManagedChannelBuilder.forTarget(platformEndpoint);
-
-        if (usePlainText) {
-            channelBuilder = channelBuilder.usePlaintext();
```

Comment:
I screwed this up a bit by duplicating code. https://github.com/opentdf/java-sdk/pull/59 should make it so that there's only one place that we create channels.

Reply:
updated